2024_Sasso_PNAS/2024_Sasso_PNAS.janno

```diff
@@ -0,0 +1,49 @@
+KOS036_MNT	F	Koksijde;WestFlanders	n/a	n/a	n/a	S41	Belgium	BE	West Flanders	Koksijde	2.64659	51.1105	contextual	n/a	n/a	n/a	650	700	750	n/a	U5b2a2b	n/a	bone_petrous	1	utigKOS036	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.5.1	99.995697	146760	0.11575912778127098	0.216732	2.077037e-15	ANGSD[v0.935]	Nr Snps (per library): 351. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB70768;ERS20173518;ERR13189848	SassoPNAS2024	KOS036	KOS036	0.011747324518671	0.001011244937803	0.8721801723123751	0.004407638701101
+KOS038_MNT	F	Koksijde;WestFlanders	n/a	n/a	n/a	S43	Belgium	BE	West Flanders	Koksijde	2.64659	51.1105	contextual	n/a	n/a	n/a	650	700	750	n/a	H1c4	n/a	tooth_root	1	utigKOS038	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.5.1	99.939865	375755	0.217593488818375	0.266317	6.351098e-13	ANGSD[v0.935]	Nr Snps (per library): 3209. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB70768;ERS20173519;ERR13189849	SassoPNAS2024	KOS038	KOS038	0.007057016815143	0.0008496057884960001	0.9082760247660701	0.008990235169119001
+KOS039_MNT	M	Koksijde;WestFlanders	n/a	n/a	n/a	S51	Belgium	BE	West Flanders	Koksijde	2.64659	51.1105	C14	RICH-26488	1269	24	650	700	750	Calibrated date using OxCal v4.4.4 Bronk Ramsey (2021), using the IntCal calibration curve from Reimer et al. 2020, doi:10.1017/RDC.2020.41	R0	R1	bone_talus	1	utigKOS039	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.5.1	99.98111	17055	0.100735886510685	n/a	n/a	n/a	Nr Snps (per library): 3. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB70768;ERS20173520;ERR13189850	SassoPNAS2024;DewildeAOE2019	KOS039	KOS039	0.025288732011124002	0.02925463208126	0.45220466139834203	0.40097125682622903
+OLV021_MNT	M	SintTruiden;Limburg	n/a	n/a	n/a	162	Belgium	BE	Limburg	Sint-Truiden	5.18857	50.82605	contextual	n/a	n/a	n/a	n/a	n/a	n/a	n/a	U4c2a	R1b	tooth_root	1	utigOLV021	Shotgun	minus	ds	haploid	https://github.com/nf-core/eager/releases/tag/2.5.1	99.976596	351358	0.13097019313035	0.0006	3.433512e-16	ANGSD[v0.935]	Nr Snps (per library): 1357. Estimate and error are weighted means of values per library. Libraries with fewer than 100 SNPs used in contamination estimation were excluded.	PRJEB70768;ERS20173528;ERR13189857	SassoPNAS2024	OLV021	OLV021	0.003696542021447	0.005104558853437	0.303579379158466	0.381360875284107
```

Comment:
The samples from Sint-Truiden and one from Wulpen don't have dating information. Then the `Date_Type` should not be set to `contextual`, but instead just to `n/a`.

Alternatively we could leave it at `contextual` and add conservatice ranges from 500 to 1500AD for Sint-Truiden ("Early to Late Medieval periods") and 1000 to 1500 for Wulpen ("High to Late Medieval (11th to 15th cc)").